distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -340,7 +340,7 @@ type File
 
              example_append = Examples.data_dir / "scratch_file"
     @subpath get_child_widget
-    / : (Text | File) -> File
+    / : Text -> File
     / self subpath = self.resolve subpath
```

Comment:
```suggestion
    / self (subpath : Text) = self.resolve subpath
```

I'd also add the inline type to ensure it is actually checked

Reply:
And I think we should probably do the same for `resolve` itself, to be consistent

Reply:
And then we could remove the second specialization of `resolve`:
![image](https://github.com/enso-org/enso/assets/1436948/5a98a12d-bc1c-4754-969e-4ded3392b250)

since it would no longer be used.